templates/tonic-notifications-deployment.yaml

```diff
@@ -122,10 +122,16 @@ spec:
           {{- end }}
         resources:
           limits:
-            memory: "1Gi"
+            memory: {{ default "1Gi" .Values.tonicai.notifications.memory.limit | quote }}
+            {{- if .Values.tonicai.notifications.cpu.limit }}
```

Comment:
I read this as "no limits for CPU unless we set some" - is that right?

Reply:
I want to submit a PR to the upstream for this as well. In the upstream, they do not set CPU requests or limits, so I followed that here. We will definitely be setting CPU requests and limits in our environment, but we'll do that through our values.